cluster/addons.go

```diff
@@ -62,6 +62,9 @@ type ingressOptions struct {
 	AlpineImage       string
 	IngressImage      string
 	IngressBackend    string
+	HTTPPort          string
+	HTTPSPort         string
+	HostNetwork       string
```

Comment:
Can we have this a bool? We can get the validation directly from the json/yaml parser itself. Also, it would be similar to how we specify things in Pod Spec.

Reply:
Changed the types.